client/src/Components/Layouts/404Page/index.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import {
+  PageNotFoundWrapper,
+  NumbersWrapper,
+  Image,
+  ZeroWrapper,
+  ZeroImage,
+  NailImg,
+  WallImage,
+  MsgWrapper,
+  MsgTitle,
+  MsgDescription,
+  Button,
+} from "./StyledComponets";
+
+import Zero from "../../../assets/zero.png";
+import Four from "../../../assets/four.png";
+import Nail from "../../../assets/nail.png";
+import Wall from "../../../assets/wall.png";
+
+export default () => (
+  <PageNotFoundWrapper>
+    <NumbersWrapper>
+      <Image src={Four} />
+      <ZeroWrapper>
+        <ZeroImage src={Zero} />
+        <NailImg src={Nail} />
+      </ZeroWrapper>
+      <Image src={Four} />
+    </NumbersWrapper>
+    <WallImage src={Wall} />
+    <MsgWrapper>
+      <MsgTitle>
+        Page Not Found!
+      </MsgTitle>
+      <MsgDescription>
+        The page you are looking for was moved, removed, renamed or might never existed.
```

Comment:
Small error in wording:

The page you are looking for was moved, removed, renamed or might **have** never existed.